src/Symfony/Component/Messenger/Transport/AmqpExt/Connection.php

```diff
@@ -62,9 +62,8 @@ public function __construct(array $connectionOptions, array $exchangeOptions, ar
     {
         $this->connectionOptions = array_replace_recursive([
             'delay' => [
-                'routing_key_pattern' => 'delay_%exchange_name%_%routing_key%_%delay%',
                 'exchange_name' => 'delay',
-                'queue_name_pattern' => 'delay_queue_%exchange_name%_%routing_key%_%delay%',
+                'queue_name_pattern' => 'delay_%exchange_name%_%routing_key%_%delay%',
```

Comment:
we need to change the name because of the new x-expire argument which would otherwise cause a precondition failed error on redeclaring. but the name is an implementation detail and should not matter.